Sources_v3/StreamChat/APIClient/Endpoints/Payloads/CurrentUserPayloads.swift

```diff
@@ -61,8 +61,46 @@ class CurrentUserPayload<ExtraData: UserExtraData>: UserPayload<ExtraData> {
 }
 
 /// An object describing the outgoing user JSON payload.
-class CurrentUserRequestBody<ExtraData: UserExtraData>: Encodable {
-    // TODO: Add more fields while working on CIS-235
+struct CurrentUserUpdateRequestBody<ExtraData: UserExtraData>: Encodable {
+    let name: String?
+    let imageURL: URL?
+    let extraData: ExtraData?
+    
+    init(name: String? = nil, imageURL: URL? = nil, extraData: ExtraData? = nil) {
+        self.name = name
+        self.imageURL = imageURL
+        self.extraData = extraData
+    }
+    
+    func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: UserPayloadsCodingKeys.self)
+        try container.encodeIfPresent(name, forKey: .name)
+        try container.encodeIfPresent(imageURL, forKey: .imageURL)
+        try extraData?.encode(to: encoder)
+    }
+}
+
+struct CurrentUserUpdateResponse<ExtraData: UserExtraData>: Decodable {
```

Comment:
Same here, this should just be `UserUpdateResponse`

Reply:
same as above